README.md

```diff
@@ -0,0 +1,50 @@
+tram
+---------------
+
+tram is a C++ networking & sockets library.
+It is developed and tested on Linux.
+tram's goal is to stay minimal in terms of dependencies, so it should be easy to port it to other platforms.
+
+# Features & Concepts
+
+- simple stream-like interface, but for sockets
+- RAII sockets management: creating socket = connecting, destroying socket = disconnecting
+
+## Getting Started
+
+tram uses familiar stream-like interface for sockets.
+
+## Connecting to server
```

Comment:
Maybe we should mention that client is disconnected when all references to its socket are destroyed